include/boost/beast/core/flat_buffer.hpp

```diff
@@ -518,7 +521,20 @@ class basic_flat_buffer
     void swap(basic_flat_buffer&, std::true_type);
     void swap(basic_flat_buffer&, std::false_type);
     char* alloc(std::size_t n);
+
+    auto data_impl(std::size_t, std::size_t) -> mutable_buffers_type;
+    auto data_impl(std::size_t, std::size_t) const -> const_buffers_type;
+    void shrink_impl(std::size_t);
+};
+
+namespace detail
+{
+template<class Allocator>
+struct is_dynamic_buffer_v0<basic_flat_buffer<Allocator>>
```

Comment:
This could go in `core/impl/flat_buffer.hpp` no?

Reply:
or even detail

Reply:
... on second thoughts, it seems pointless making 5 new files for this

Reply:
Yeah 5 new files is unnecessary, but core/impl/flat_buffer.hpp already exists why not put it there

Reply:
I was not clear. I agree with your suggestion.